### DISCUSSION TITLE
Background relative path

### DISCUSSION THREAD

**Discussion by Glyphack** [wezterm/wezterm#4526] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20230712-072601-f4abf8fd

### Ask your question!

Is it possible to specify relative path or use ~ when specifying the background image? I tried both of them and it does not work but putting the absolute path works fine.

**Comment by nekowinston** [comment, marked as answer]
You can use [`wezterm.home_dir`](https://wezfurlong.org/wezterm/config/lua/wezterm/home_dir.html) instead of `~`: `wezterm.home_dir .. "/your/relative/path.png"`
